lib/resin-lint.ts

```diff
@@ -256,6 +268,7 @@ export const lint = (passedParams: any) =>
 				const prettierCheck = options.argv.prettier !== false;
 				const testsCheck = options.argv.tests === true;
 				const typescriptCheck = options.argv.typescript;
+				const autoFix = options.argv.fix;
```

Comment:
```suggestion
				const autoFix = options.argv.fix === true;
```

like above. In fact, I wonder what `optimist` does for non-provided boolean options -- is the option `undefined`?